python/lsst/ts/electrometer/commander.py

```diff
@@ -119,21 +129,33 @@ async def send_command(
         -------
         reply
         """
-
+        if timeout is None: 
+            self.log.debug(f"Will use timeout {self.timeout}s")
+        else:
+            self.log.debug(f"Will use timeout {timeout}s")
```

Comment:
do you need to set `self.timeout=timeout` here?

Reply:
You either define the timeout or use self.timeout, the default. I was having trouble with the timeouts not being long enough, so I added this debugging sequence. It could potentially be removed at a later date, but I think it's nice to leave in for now.